### ISSUE TITLE
add all metrics at notebook #758 (Migrated Issue)

### ISSUE THREAD

**Issue by tahalpara** [sapientml/sapientml#57]
**Migration of Issue from the old repo https://github.com/F-AutoML/sapientml/issues/758**

Please note following comments by

[mariko-sugawara](https://github.com/mariko-sugawara)(msugawara) commented [on Apr 20]

**Is your feature request related to a problem? Please describe.**
AsIs:

#EVALUATION
if set(_TARGET_COLUMNS).issubset(__test_dataset.columns.tolist()):
    from sklearn.metrics import mean_squared_error
    __rmse = mean_squared_error(__target_test, __y_pred, squared=False)
    print('RESULT: RMSE:', str(__rmse))

**Describe the solution you'd like**
This is useful for users when the script and notebook(final_script.py and final_script.py.ipynb) display all metrics instead of just one.

Following were the relevant updates

@[mariko-sugawara](https://github.com/mariko-sugawara) mariko-sugawara added the [enhancement](https://github.com/F-AutoML/sapientml/labels/enhancement) label [on Apr 20](https://github.com/F-AutoML/sapientml/issues/758#event-9064004664)
@[mariko-sugawara](https://github.com/mariko-sugawara) mariko-sugawara added this to [SapientML Development backlog](https://github.com/orgs/F-AutoML/projects/2) [on Apr 24](https://github.com/F-AutoML/sapientml/issues/758#event-922e858c-f271-5ba8-809f-bf7f77aaecf5)
@[github-project-automation](https://github.com/apps/github-project-automation) github-project-automation bot moved this to 🆕 New in [SapientML Development backlog](https://github.com/orgs/F-AutoML/projects/2) [on Apr 24](https://github.com/F-AutoML/sapientml/issues/758#event-b9664d73-41b6-50a9-ac66-0666b82cd804)

**Comment by kimusaku** [comment]
@tahalpara
Thanks for the issue, but it looks this does not fit to the current status. I will once close it, and please create a new one.
